### ISSUE TITLE
Unable to load image: amlightrain.png'

### ISSUE THREAD

**Issue by derwegi** [davidrlunu/dots-and-dashes#5]
I tried to get the weather running and got this message:

conky: Unable to load image '/home/user/.config/conky/weather/weather_icons/day/amlightrain.png'. 

The file seems to be missing in your git repo.

**Comment by davidrlunu** [comment]
Yes, it might be! Unfortunately I couldn't cover all the weather conditions as weather. com does not have a spreadsheet of available conditions. 
I will update the repo with the website to download the weather icons, in order to expand the cases. 

Thank you for the note!

**Comment by jspaezp** [comment]
same case for "am clouds / pm sun" which could be a little bit .. "buggier" due to the /

ps: exelent work, thanks for sharing

**Comment by davidrlunu** [comment]
unfortunatelly at the moment i created the script i couldn't predict every
weather condition and weather.com does not have a database of all weather
conditions. It happens to me at well (to the day) that i still need to add
forecast conditions that i was not aweare of.

About the "/" thing i know from experience that ruins everything. In my
case i only encountered "/rain"

Sorry for incomplete work but it was not in my power :C

2016-10-18 0:56 GMT+02:00 jspaezp notifications@github.com:
